src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java

```diff
@@ -313,7 +323,7 @@ private static String createPropertiesText() {
                 String typeText = "int[]";
                 if (!property.get(2).getTextContent().contains("subset of tokens")) {
                     final String typeName =
-                            property.get(2).getFirstChild().getFirstChild().getTextContent();
+                            property.get(2).getFirstChild().getFirstChild().getTextContent().trim();
```

Comment:
I guess same changes has been added in other PR. Please rebase

Reply:
Done